src/applications/find-va-forms/api/index.js

```diff
@@ -23,14 +23,8 @@ export const fetchFormsApi = async (query, options = {}) => {
     response = await apiRequest(FORMS_URL);
   }
 
-  // Derive the forms.
   const forms = response?.data;
-
-  // Give back the normalized forms data.
-  const normalizedForms = normalizeFormsForTable(forms);
-
-  // Sort the forms by 'id' and 'asc' by default.
-  const sortedForms = sortBy(normalizedForms, 'id');
+  const sortedForms = sortBy(forms, 'id');
```

Comment:
I removed the formatter because reshaping the data isn't really necessary now without the table component.